package.json

```diff
@@ -6,21 +6,22 @@
   "directories": {
     "doc": "docs"
   },
-  "dependencies": {
+  "dependencies": {},
+  "devDependencies": {
+    "@fortawesome/fontawesome-free": "^5.2.0",
```

Comment:
is `fortawesome` right here?

Reply:
yes: https://www.npmjs.com/package/@fortawesome/fontawesome-free